OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/LoginPrefs.kt

```diff
@@ -168,7 +172,9 @@ class LoginPrefs @Inject constructor(
         private const val PROFILE_IMAGE = "profile_image"
         private const val AUTH_TOKEN_SOCIAL = "facebook_token"
         private const val AUTH_TOKEN_BACKEND = "google_token"
+        //This key is used to store the Auth type, either password or any social provider
         private const val ANALYTICS_KEY_BACKEND = "segment_backend"
+        private const val ALREADY_REGISTERED_LOGGED_IN = "already_registered_logged_in"
```

Comment:
What about `ALREADY_REGISTERED_AND_LOGGED_IN`? This conveys more information. 

And we should update the relative names acordingly.